solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -369,6 +378,77 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         else store.removePoolFromWhitelist(_poolAnchor);
     }
 
+    /**
+     * @dev adds a high tier pool
+     * can only be called by the contract owner
+     *
+     * @param _poolAnchor pool anchor
+     */
+    function addHighTierPool(IConverterAnchor _poolAnchor)
```

Comment:
Perhaps it's worth to add add/remove events for this (or since it's owner-only, it's not important)?

Reply:
yeah I thought it's not important for now since it's a temporary solution - once we have the full tier system, we'll have events for all of these (we need to add these pools to the high tier list with every upgrade for now)

Reply:
👍